lib/hard_data.dart

```diff
@@ -21,7 +28,7 @@ const List<ReusableCard> staffWelfare = [
     routeTo: Routes.ppe,
   ),
   ReusableCard(
-    title: 'Your Welfare',
+    title: Strings.yourWelfareTitle,
     description: 'Vital info & contacts',
```

Comment:
I wonder if it makes sense to extract the titles, but not the descriptions? I think it's fine with either both in strings, or both here.

Reply:
Yeah .... was up and down on that too. In `hard_data.dart` there are lots of these choices.

In this case the title is used multiple places in code so makes sense ie. 'Your Welfare' in the cared title on the Home screen, but also on the AppBar within that screen. So makes sense to keep those two in sync.

I left the description as only used once. So either would be fine I guess. But for example for ALL the strings in `hard_data.dart` I don't think we want to move them to Strings, as almost no advantage vs leave where they are. 

So deferred these choices for now. But at least its just these 2 files involved. So much easier to chnage this later.